codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/EnumGenerator.kt

```diff
@@ -198,14 +209,28 @@ open class EnumGenerator(
         }
     }
 
+    private fun renderUnknownVariantValue() {
+        // No doc or note comes with this inner opaque struct.
+        // We do want to mark it as #[allow(missing_docs)] to suppress the missing docs lint.
+        writer.docWithNote(null, null)
```

Comment:
remove prbably

Reply:
we probably want docs right? explaining that it shouldn't be used?

Reply:
Yes we want docs, but it's more for the whole enum as opposed to just the `Unknown` variant. What the code is really trying to achieve here is to add `#[allow(missing_docs)]` for the `Unknown` variant, but calling `writer.docWithNote(null, null)` might have been roundabout. Replaced by `rust("##[allow(missing_docs)]")` in 99a645f.